integration-api/src/main/java/se/inera/intyg/minaintyg/integration/api/certificate/dto/Certificate.java

```diff
@@ -0,0 +1,24 @@
+package se.inera.intyg.minaintyg.integration.api.certificate.dto;
```

Comment:
Are these files in the right place?

Reply:
@mww-sopra-steria I was unsure of where to place many files so please let me know if I should move some of them